appserver/security/core-ee/src/main/java/com/sun/enterprise/security/ee/SecuritySniffer.java

```diff
@@ -80,11 +81,27 @@
     };
 
     public SecuritySniffer() {
-        super("security", "WEB-INF/web.xml", null);
-        
+        super("security", null, null);
     }
-    
-   /**
+
+    /**
+     * Returns true if the passed file or directory is recognized by this
+     * sniffer.
+     *
+     * @param context deployment context
+     * @return true if the location is recognized by this sniffer
+     */
+    public boolean handles(DeploymentContext context) {
```

Comment:
Can you please add an @Override annotation here? It's a bit of nitpicking, but it is good practice to do this so the compiler can catch any typo in the intended overridden method when the method signatures does not match.